### ISSUE TITLE
Error in running getdata.sh ?

### ISSUE THREAD

**Issue by anubhavj880** [BIDData/BIDMach#163]
When i run getdata.sh file to download all the dataset in data folder so after data all data download a error comes like /tparse.exe : No such file or directory and same for trec.exe so i want to know where is this two files. Please explain

**Comment by kpengboy** [comment]
Did you run `mvn package` first?

**Comment by anubhavj880** [comment]
Yes I run the mvn package and its also downloading some file of approx 30 GB but after that its stops because this types of error comes like
/BIDMach/scripts/../cbin/trec.exe: No such file or directory
/BIDMach/scripts/../cbin/tparse.exe: No such file or directory
And i have searched there is no folder inside scripts with name cbin which contain trec.exe or tparse.exe

**Comment by jcanny** [comment]
The path is actually (note the /../) /BIDMach/cbin/trec.exe and tparse.exe

Those files should be installed by 
"mvn install"

btw you should always report the platform you are using so we can better diagnose what might be wrong.

**Comment by anubhavj880** [comment]
I am using windows 10.

Sent from Mail for Windows 10
